startblog/models.py

```diff
@@ -1,3 +1,18 @@
 from django.db import models
-
+from django.contrib.auth.models import User
 # Create your models here.
+
+class Blogs(models.Model):
+    bloger = models.ForeignKey(User,on_delete=models.CASCADE)
+    coverpic = models.ImageField(upload_to='pics')
+    title = models.CharField(unique=True,max_length=100)
+    content = models.CharField(max_length=1000)
```

Comment:
for larger strings you can use models. TextField

Reply:
sure. Thanx